doc/source/whatsnew/v1.3.0.rst

```diff
@@ -192,6 +192,7 @@ Categorical
 - Bug in ``CategoricalIndex.reindex`` failed when ``Index`` passed with elements all in category (:issue:`28690`)
 - Bug where constructing a :class:`Categorical` from an object-dtype array of ``date`` objects did not round-trip correctly with ``astype`` (:issue:`38552`)
 - Bug in constructing a :class:`DataFrame` from an ``ndarray`` and a :class:`CategoricalDtype` (:issue:`38857`)
+- Bug where :class:`DataFrame` axes with an empty :class:`CategoricalIndex` could not be reindexed if the target contained duplicates (:issue:`38906`)
```

Comment:
- Bug in :meth:`DataFrame.reindex` with :class:`CategoricalIndex`  was throwing `IndexError`for frame with an empty index when new index contains duplicates (:issue:`38906`)

Reply:
I think this wording is slightly misleading because it makes it sound like the problem arises when the argument to `reindex` is a `CategoricalIndex`, when in fact the problem occurs when the DataFrame axes are categorical. I've tried to rephrase to avoid this potential confusion.